test_runner/src/main/kotlin/ftl/args/yml/FlankYml.kt

```diff
@@ -2,24 +2,35 @@ package ftl.args.yml
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.annotation.JsonProperty
+import ftl.config.FtlConstants.GCS_PREFIX
 import ftl.util.Utils.fatalError
 
 /** Flank specific parameters for both iOS and Android */
 @JsonIgnoreProperties(ignoreUnknown = true)
 class FlankYmlParams(
     val testShards: Int = 1,
     val repeatTests: Int = 1,
+    val junitGcsPath: String = "",
 
     @field:JsonProperty("test-targets-always-run")
     val testTargetsAlwaysRun: List<String> = emptyList()
 ) {
     companion object : IYmlKeys {
-        override val keys = listOf("testShards", "repeatTests", "test-targets-always-run")
+        override val keys = listOf("testShards", "repeatTests", "junitGcsPath", "test-targets-always-run")
```

Comment:
Question unrelated to PR.

What's this?

Reply:
`IYmlKeys` is used by the `flank doctor` command to detect unregonized YAML keys.

https://github.com/TestArmada/flank/blob/5715cd44bf784d2b5508d766f01c30aea7bbd26d/test_runner/src/main/kotlin/ftl/doctor/Doctor.kt#L37